tests/test_dagmc_stats.py

```diff
@@ -8,7 +8,7 @@
 import unittest
 
 entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
-test_env = [ { 'input_file' : 'tests/3vols.h5m'}, {'input_file' : 'tests/single-cube.h5m'} ]
+test_env = [ { 'input_file' : 'tests/3vols.h5m'}, {'input_file' : 'tests/single-cube.h5m'}, {'input_file' : 'tests/pyramid-2.h5m'}]
```

Comment:
the `test_get_lri()` function should use this new test geometry. I don't think it makes sense for us to use the cube when all the values are zero (I think that is also why the test is failing in python 3).

Reply:
also this file can just be called `pyramid.h5m` since there is not a pyramid-1 anywhere.